src/test/java/uk/gov/hmcts/dts/fact/services/CourtServiceTest.java

```diff
@@ -501,4 +501,14 @@ void shouldReturnEmptyListIfNoServiceArea() {
         assertThat(results.getCourts()).isNull();
         verifyNoInteractions(serviceAreaSearchFactory);
     }
+
+    @Test
+    void shouldReturnCourtReferenceListWhenSearchingByPrefixAndActive() {
+        final Court court = mock(Court.class);
+        when(courtRepository.findCourtBySlugStartingWithAndDisplayed(anyString(), anyBoolean()))
+            .thenReturn(singletonList(court));
+        final List<CourtReference> results = courtService.getCourtsByPrefixAndActiveSearch("mosh kupo", true);
+        assertThat(results.size()).isEqualTo(1);
```

Comment:
Is is cleaner to use assertThat(results).hasSize(1) since that is provided by assertJ